app/src/pages/inside/dashboardItemPage/modals/editWidgetModal/editWidgetModal.jsx

```diff
@@ -132,6 +133,12 @@ export class EditWidgetModal extends Component {
     } = this.props;
 
     const data = prepareWidgetDataForSubmit(this.preprocessOutputData(widgetSettings));
+    const isForceUpdated =
```

Comment:
Maybe `isForceUpdateNeeded`, because we need to determine if an update is needed before taking action.